src/Explorer/Controls/Settings/SettingsComponent.tsx

```diff
@@ -426,7 +426,21 @@ export class SettingsComponent extends React.Component<SettingsComponentProps, S
             throw error;
           }
         } else {
-          const updatedOffer: DataModels.Offer = await updateOffer(this.collection.offer(), newOffer, headerOptions);
+          const updateOfferParams: DataModels.UpdateOfferParams = {
+            databaseId: this.collection.databaseId,
+            collectionId: this.collection.id(),
```

Comment:
have you tested this using the feature flag "enableSettingsV2" ?
just to be sure, make sure you can update from autopilot -> manual and manual -> autopilot when you have the feature flag set

Reply:
Tested this change with the new settings tab and it worked fine.